cluster-essentials/pkg/client/fakes.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package client
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cp "github.com/otiai10/copy"
+)
+
+func DownloadBundleSuccess(bundlePath, outputDir string) error {
+	outputDir = filepath.Join(outputDir, "package")
+	_ = os.MkdirAll(outputDir, os.ModePerm)
+	_ = cp.Copy("../testdata/package2", outputDir)
+	return nil
+}
+
+func DownloadBundlesWithEmptyBundle(bundlePath, outputDir string) error {
+	return nil
+}
+
+func DownloadCorruptedBundleSuccess(bundlePath, outputDir string) error {
+	outputDir = filepath.Join(outputDir, "package")
+	_ = os.MkdirAll(outputDir, os.ModePerm)
+	_ = cp.Copy("../testdata/package1", outputDir)
+	return nil
+}
+
+func DownloadBundleFail(bundlePath, outputDir string) error {
+	return fmt.Errorf("image not found")
+}
```

Comment:
Please un-export these functions. Ideally, they can live in `install_test.go` since that's the only place it's used.

Reply:
+1